rosetta/text/vw_helpers.py

```diff
@@ -71,6 +73,10 @@ def parse_lda_topics(topics_file, num_topics, normalize=True):
         The --readable_model output of a VW lda run
     num_topics : Integer
         The number of topics in every valid row
+    max_token_hash : Integer
```

Comment:
Why would someone want to truncate hashed values above some number?  Seems like a funny way to cut things off.